charts/ozone/templates/om/om-service-headless.yaml

```diff
@@ -28,6 +28,10 @@ spec:
   ports:
     - name: ui
       port: {{ .Values.om.service.port }}
+    {{- if gt (int .Values.om.replicas) 1 }}
+    - name: ratis
+      port: 9872
+    {{- end }}
```

Comment:
As per [OM documentation](https://ozone.apache.org/docs/current/feature/om-ha.html)
>This logical name is called serviceId and can be configured in the ozone-site.xml
>The defined serviceId can be used instead of a single OM host using [client interfaces](https://ozone.apache.org/docs/current/interface.html)

Perhaps there should be a different service which maps and groups pods as per the `serviceId`.

Reply:
> Is this port being exposed for one Ozone Manager(OM) to communicate with another OM when they form a Ratis ring?

yes this was my reason

> My understanding is that the current selector will match this service with all OM pods. Consequently, the messages sent to this service will be forwarded to a random OM pod instead of a specific OM pod.

ok if this is the case maybe we can remove this export. I found and used the following Ticket for the port configuration: https://issues.apache.org/jira/browse/HDDS-4677. I'm not really familiar with the architecture. If we can remove it I will do so :)

Reply:
> If the number of OM pods are manually modified by kubectl scale then this port will perhaps never be exposed. We should think if there is a downside to always exposing the port. 

Great point! I hadn't considered that scenario. What could be the downside of exposing the port within an internal Kubernetes network? Perhaps we can use a Helm lookup mechanism to check the current replica count as an alternative, but the simplest approach is to always expose the port.